### PR TITLE
Vk serialization fix

### PR DESCRIPTION
**Pull request by AntoineRondelet**

### CODE REVIEW

File: libzeth/tests/serialization/proto_utils_test.cpp [hunk 1]
```diff
@@ -49,7 +49,24 @@ TEST(ProtoUtilsTest, PrimaryInputsEncodeDecode)
...
+    std::vector<G1> rest{G1::random_element(), G1::random_element()};
+    acc_vect.rest = libsnark::sparse_vector<G1>(std::move(rest));
+
+    std::string acc_vect_string =
+        libzeth::accumulation_vector_to_string<ppT>(acc_vect);
+
+    const libsnark::accumulation_vector<G1> acc_vect_decoded =
+        libzeth::accumulation_vector_from_string<ppT>(acc_vect_string);
+
+    std::string acc_vect_string_recovered =
```
Comment (AntoineRondelet) [inline libzeth/tests/serialization/proto_utils_test.cpp:65]: This variable does not seem to be used anywhere

File: libzeth/core/group_element_utils.tcc [hunk 2]
```diff
@@ -20,6 +20,28 @@ std::string point_g1_affine_to_json(const libff::G1<ppT> &point)
            "\"]";
 }
 
+template<typename ppT>
+libff::G1<ppT> point_g1_affine_from_json(const std::string &grp_str)
```
Comment (dtebbs) [inline libzeth/core/group_element_utils.tcc:24]: tested via other functions, but ideally should have its own test.

File: libzeth/tests/serialization/proto_utils_test.cpp (hunk 1 above)
Comment (dtebbs) [reply libzeth/tests/serialization/proto_utils_test.cpp]: cleaned the function

File: libzeth/core/group_element_utils.tcc (hunk 2 above)
Comment (dtebbs) [reply libzeth/core/group_element_utils.tcc:24]: added test

**Pull request merged**